analysis/model/04_01_(a)_cox_fit_model.R

```diff
@@ -205,15 +205,15 @@ coxfit <- function(data_surv, interval_names, covar_names, mdl, subgroup,non_cas
     
     if(non_case_inverse_weight ==1){
       print("Using regular SE's for CI's")
-      results$conf.low=exp(confint(fit_cox_model,level=0.95)[,1]) #use robust standard errors to calculate CI
+      results$conf.low=exp(confint(fit_cox_model,level=0.95)[,1]) #use regular standard errors to calculate CI for non-sampled data
       results$conf.high=exp(confint(fit_cox_model,level=0.95)[,2])
     }else if(non_case_inverse_weight !=1){
       print("Using robust SE's for CI's")
-      results$conf.low=exp(confint(robust_fit_cox_model,level=0.95)[,1]) #use robust standard errors to calculate CI
+      results$conf.low=exp(confint(robust_fit_cox_model,level=0.95)[,1]) #use robust standard errors to calculate CI for sampled data
       results$conf.high=exp(confint(robust_fit_cox_model,level=0.95)[,2])
     }
-    results$std.error=exp(sqrt(diag(vcov(fit_cox_model))))
-    results$robust.se=exp(sqrt(diag(vcov(robust_fit_cox_model))))
+    results$std_error_ln_hr=sqrt(diag(vcov(fit_cox_model)))
```

Comment:
For consistency, please can this be called `se_ln_hr` to match `robust_se_ln_hr`?

Reply:
eb8be00f158352b7d593686c4d338beb892552f5